index.yaml

```diff
@@ -25,8 +25,8 @@ tools:
   slack:
     reference: ./slack
     all: true
-  linkedin:
-    reference: ./linkedin
+  linkedin-publishing:
+    reference: ./linkedin-publishing
```

Comment:
Nit:  it would be better organzied if we put folder like ./linkedin/publishing so it other linkedin tools can go to subfolder.